L.Control.Geonames.js

```diff
@@ -134,12 +140,12 @@ L.Control.Geonames = L.Control.extend({
             li.innerText = 'No results found';
         }
     },
-    _getName: function(geoname){
+    _getName: function (geoname) {
```

Comment:
This isn't working right for multiple codes.  It is currently sending them through as a comma-delimited list, but per the API, these need to have one query parameter for each code:
`...&adminCode1=OR&adminCode1=WV`